tools/build_chain.sh

```diff
@@ -1625,8 +1634,8 @@ done
 # clean
 for line in ${ip_array[*]};do
     ip=${line%:*}
-    set_value ${ip//./}_count 0
-    set_value "${ip//./}_port_offset" 0
+    set_value ${ip//[\.:]/_}_count 0
```

Comment:
Codacy found an issue: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/gh/FISCO-BCOS/FISCO-BCOS/pullRequest?prid=5837713)